django/db/models/base.py

```diff
@@ -5,6 +5,8 @@
 import warnings
 from itertools import chain
 
+from six.moves import copyreg
```

Comment:
The master branch supports Python 3 only so there shouldn't be a need to use six.

Reply:
Is this eligible to be backported to 1.11.x?